scripts/utils.ts

```diff
@@ -175,18 +36,21 @@ export async function updateApis(
  */
 export async function downloadLatestApis(
```

Comment:
This is the only method in utils.ts that is used by the oas version of `updateApis`. Every other function here has been removed as they were only used by the raml generator.